example/migrate/kube-scheduler.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  creationTimestamp: null
+  labels:
+    component: kube-scheduler
+    tier: control-plane
+  name: kube-scheduler
+  namespace: kube-system
+spec:
+  containers:
+  - command:
+    - kube-scheduler
+    - --authentication-kubeconfig=/etc/kubernetes/scheduler.conf
+    - --authorization-kubeconfig=/etc/kubernetes/scheduler.conf
+    - --bind-address=127.0.0.1
+    - --config=/var/lib/scheduler/scheduler-config.yaml
+    - --kubeconfig=/etc/kubernetes/scheduler.conf
+    - --leader-elect=true
+    - -v=5
+    image: k8s.gcr.io/kube-scheduler:v1.24.2
```

Comment:
Please add this file to the list in the following file.
(The list indicates which files should be modified when we upgrade the Kubernetes version.)
https://github.com/topolvm/topolvm/blob/main/docs/maintenance.md#kubernetes

Reply:
Added.